openedx/core/djangoapps/programs/signals.py

```diff
@@ -21,117 +23,59 @@
 @receiver(COURSE_CERT_AWARDED)
 def handle_course_cert_awarded(sender, user, course_key, mode, status, **kwargs):  # pylint: disable=unused-argument
     """
-    If programs is enabled and a learner is awarded a course certificate,
-    schedule a celery task to process any programs certificates for which
-    the learner may now be eligible.
+    If use of the Credentials IDA is enabled and a learner is awarded a course certificate, schedule a celery task to
+    determine if the learner is also eligible to be awarded any program certificates.
 
     Args:
-        sender:
```

Comment:
why did you remove this from the docstring?

Reply:
I can't remember tbh, I think I just lost it when I was doing some other work. I'll add it back in!

Reply:
Ah, I originally removed it because it was an unused argument and I thought it wasn't important for the general docstring. I've added it to each docstring though!

Reply:
thanks! I feel like as long as there are docstrings they should be complete, but I see your point. Thank you for adding it back.